resources/public/api/conf/1.0/application.raml

```diff
@@ -66,3 +66,15 @@ uses:
             type: string
             example: "X234567891BY-AT3456"
         get: !include retrieveAllocations.raml
+    /{taxYear}:
+      uriParameters:
+        taxYear:
+          description: "A unique identifier for the tax year. <br><br>Must conform to the regular expression: `^2[0-9]{3}-[0-9]{2}$`"
+          type: string
+          example: 2021-22
+      /collection:
+        (annotations.group):
+          name: Collection
+          description: ""
```

Comment:
The overview name and text for the coding out section should be based on the description found here https://confluence.tools.tax.service.gov.uk/display/MTE/Underpayments+and+Debts

Reply:
The 2nd paragraph should also be included